src/RollupGreeter.sol

```diff
@@ -11,6 +12,11 @@ import {GlobalGreeter} from "./GlobalGreeter.sol";
  * @dev This contract allows users to transmit greetings to a greeter contract deployed on the global chain via cross-chain communication.
  */
 contract RollupGreeter is XApp {
+    /**
+     * @notice Gas limit used for a cross-chain greet call
+     */
+    uint64 public constant GAS_LIMIT = 200_000;
```

Comment:
Can you go lower? and maybe name it something that says what it's for, but ekeeping it shoer